draft-ietf-quic-transport.md

```diff
@@ -4010,14 +3903,6 @@ language from Section 3 of {{!TLS13=RFC8446}}.
    } TransportParameter;
 
    struct {
-      select (Handshake.msg_type) {
-         case client_hello:
-            QuicVersion initial_version;
-
-         case encrypted_extensions:
-            QuicVersion negotiated_version;
-            QuicVersion supported_versions<4..2^8-4>;
-      };
       TransportParameter parameters<0..2^16-1>;
    } TransportParameters;
```

Comment:
This can now be a straight typedef: `TransportParameter TransportParameters<0..2^16-1>;`